### PR TITLE
Handle 'data' field in the results returned by fetchCoverage (followup of issue #1098 on services)

### PR DESCRIPTION
**Pull request by calixteman**

### CODE REVIEW

File: dxr-common.js [hunk 1]
```diff
@@ -37,15 +37,17 @@ async function getCoverage(revPromise, path) {
...
-  for (let [l, c] of Object.entries(result)) {
-    const line_no = document.getElementById(lineNoMap(l));
-    const line = document.getElementById(`line-${l}`);
-    if (c > 0) {
-      line_no.style.backgroundColor = 'greenyellow';
-      line.style.backgroundColor = 'greenyellow';
-    } else {
-      line_no.style.backgroundColor = 'tomato';
-      line.style.backgroundColor = 'tomato';
+  if (result.hasOwnProperty('data')) {
```
Comment (marco-c) [inline dxr-common.js]: Let's throw an exception `if (!result.hasOwnProperty('data'))`, so we can leave the rest of the code almost as is

File: socorro.js [hunk 2]
```diff
@@ -51,13 +51,14 @@ if (Object.keys(fileinfo).length != 0) {
         le.element.append(e);
         le.element = e;
       }
-      fetchCoverage(revision, filename).then(data => {
-        if (data !== null && !data.hasOwnProperty("error")) {
+      fetchCoverage(revision, filename).then(data => {console.log(data);
+        if (data !== null && !data.hasOwnProperty("error") && data.hasOwnProperty("data")) {
+          const covData = data["data"];
```
Comment (marco-c) [inline socorro.js]: Same here (regarding the exception)

File: socorro.js [hunk 3]
```diff
@@ -51,13 +51,14 @@ if (Object.keys(fileinfo).length != 0) {
         le.element.append(e);
         le.element = e;
       }
-      fetchCoverage(revision, filename).then(data => {
-        if (data !== null && !data.hasOwnProperty("error")) {
+      fetchCoverage(revision, filename).then(data => {console.log(data);
```
Comment (marco-c) [inline socorro.js]: There's an unintended `console.log` left here

Comment (calixteman) [reply socorro.js]: oh you see how I'm debugging ;)

**Review by marco-c — APPROVED** [review]

**Pull request merged**